clonevirtualenv.py

```diff
@@ -1,13 +1,16 @@
+#!/usr/bin/env python
 from __future__ import with_statement
 import logging
 import optparse
 import os
+import os.path
 import re
 import shutil
 import subprocess
 import sys
+import itertools
 
-version_info = (0, 2, 4)
+version_info = (0, 2, 6)
```

Comment:
I took the liberty of bumping the version. I choose 0.2.6 because the official pip release is 0.2.5